.eslintrc.json

```diff
@@ -0,0 +1,69 @@
+// Based on Odoo's .eslintrc.js
+{
+  "extends": ["eslint:recommended", "plugin:prettier/recommended", "plugin:node/recommended"],
+  "parserOptions": {
+    "ecmaVersion": 2022,
+    "sourceType": "module"
+  },
+  "env": {
+    "node": true,
+    "browser": false,
```

Comment:
isn't node or browser differently true or false (or sometimes both at the same time) depending on the file?

Reply:
the overrides below are setting the env with the right values for the few files where it is not (just) node